FluentTc/Engine/BuildConfigurationRunner.cs

```diff
@@ -76,16 +76,24 @@ private static string CreateTriggerBody(string buildConfigId, int? agentId, List
         {
             var bodyBuilder = new StringBuilder();
 
-            if (moreOptions != null &&
-                moreOptions.TriggeringOptions.Personal == true)
-            {
-                bodyBuilder.Append(@"<build personal=""true"">").AppendLine();
-            }
-            else
+            bodyBuilder.Append(@"<build");
+
+            if (moreOptions != null)
             {
-                bodyBuilder.Append(@"<build>").AppendLine();
+                if (moreOptions.TriggeringOptions.Personal == true)
+                {
+                    bodyBuilder.Append(@" personal=""true""");
+                }
+
+                if (moreOptions.GetBranchName() != null)
```

Comment:
I assume that you meant to write is "moreOptions.HasBranch()" instead of "moreOptions.GetBranchName() != null"